src/components/login/LoginPage.tsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import {StyleSheet, View, TextInput, Text, TouchableOpacity, Alert, Image, Pressable} from 'react-native';
+import { email, eye, login } from '../../../assets/images';
+
+const LoginPage = () => {
+  const [text, onChangeText] = React.useState(null);
+  const [passwordSecured, setPasswordSecured] = React.useState(true);
+
+  return (
+    <View style={styles.containerBox}>
+        <View style={styles.containerImage}>
+          <Image resizeMode='contain' style={styles.imageLogin} source={login} />
+        </View>
+        <View style={styles.inputView}>
+          <Image resizeMode='contain' style={styles.imageInput} source={email} />
+          <TextInput
+            style={styles.input}
+            onChangeText={onChangeText}
+            value={text}
+            placeholder={"Enter your Email"}
+            placeholderTextColor={"#a8a8a8"}
+          />  
+        </View>
+        <View style={styles.inputView}>
+          <TextInput
+            style={styles.input}
+            secureTextEntry={passwordSecured}
+            textContentType='password'
+            placeholder={"Password"}
+            placeholderTextColor={"#a8a8a8"}
+          />        
+          <TouchableOpacity
+            style={styles.passwordIcon}
+            onPress={() => {
+              setPasswordSecured(!passwordSecured);
+            }}>
+            <Image resizeMode='contain' style={styles.imageInput} source={eye} />
+          </TouchableOpacity>
+        </View>
+        <TouchableOpacity
+          style={styles.buttonBlockLogin}
+          onPress={() => {
+            Alert.alert(
+              'Login pressed'
+            );
+          }}>
+          <Text style={styles.buttonTextLogin}>
+            Login
+          </Text>
+        </TouchableOpacity>
+        <Pressable children={({ pressed }) => (
+          <Text style={{  textAlign: 'center', padding:5,marginTop:15,color: pressed ? '#444' : '#1e6cab'}}>
+            Don't have an account? Register Now
+          </Text>
+        )}/>
+        <Pressable children={({ pressed }) => (
+          <Text style={{  textAlign: 'center', padding:5,color: pressed ? '#444' : '#1e6cab'}}>
+            Forgot Password
+          </Text>
+        )}/>
+      </View> 
+  );
+};
+
+
+const styles = StyleSheet.create({
```

Comment:
Also, as @AND-GORNIY said on last lecture, avoid SUPER MEGA components. SOLID principles, first letter S means self-responsibility. Move common components and logic outside.